test/e2e/pkg/testnet.go

```diff
@@ -49,17 +49,20 @@ const (
 
 // Testnet represents a single testnet.
 type Testnet struct {
-	Name             string
-	File             string
-	Dir              string
-	IP               *net.IPNet
-	InitialHeight    int64
-	InitialState     map[string]string
-	Validators       map[*Node]int64
-	ValidatorUpdates map[int64]map[*Node]int64
-	Nodes            []*Node
-	KeyType          string
-	ABCIProtocol     string
+	Name                   string
+	File                   string
+	Dir                    string
+	IP                     *net.IPNet
+	InitialHeight          int64
+	InitialState           map[string]string
+	Validators             map[*Node]int64
+	ValidatorUpdates       map[int64]map[*Node]int64
+	Nodes                  []*Node
+	KeyType                string
+	ABCIProtocol           string
+	PrepareProposalDelayMS int
+	ProcessProposalDelayMS int
+	CheckTxDelayMS         int
```

Comment:
Why not just use durations directly for the type? 
Additionally, consider using a struct to contain these. 

``` 
type ABCIDelays struct {
PrepareProposal
...
}
```

Reply:
Does TOML support durations or are you suggesting I just convert it after parsing the toml file?

Reply:
Our toml parser supports parsing duration strings into go durations. 

see https://github.com/tendermint/tendermint/blob/cbc7a1abcfe871cc0122ee89b841b3803f9478a9/docs/tendermint-core/configuration.md#L344

Reply:
Let me change it up then